src/apply.jl

```diff
@@ -75,9 +73,14 @@ function apply(table, t::Transform; cols=_get_cols(table), header=nothing, kwarg
     # Extract a columns iterator that we should be able to use to mutate the data.
     # NOTE: Mutation is not guaranteed for all table types, but it avoid copying the data
     coltable = Tables.columntable(table)
-    cols = _to_vec(cols)
-
-    result = reduce(hcat, [_apply(getproperty(coltable, col), t; kwargs...) for col in cols])
+    # Combine the Vector{Vector} of components into a Matrix
```

Comment:
```suggestion
    # Combine the Vector{Vector} of components into an Array, which might still be a Vector
```

Reply:
Is it because you can get e.g. `[[1], [2], [3]]` which is a `Vector{Vector}` but becomes a `Vector` as `[1, 2, 3]`?